tasks/setup-zypper.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: If choosing custom package set, ensure minimal cockpit is included
+  set_fact:
+    cockpit_packages: "{{ cockpit_packages + __cockpit_packages_minimal
+      if cockpit_packages not in __cockpit_package_types
+      else cockpit_packages }}"
+  when: cockpit_packages is defined
+
+- name: Ensure Cockpit Web Console packages are installed.
+  zypper:
+    name: "{{ __cockpit_packages[cockpit_packages]
+      if cockpit_packages in __cockpit_package_types
+      else cockpit_packages }}"
+    state: present
+
+- name: Reboot system
+  reboot:
+  when: ansible_distribution == 'ALP-Dolomite'
+
+- name: Flush handlers
```

Comment:
If you reboot the system, do you still need to run the handlers?
```
# handlers file for cockpit
- name: Reload systemd
  systemd:
    daemon_reload: true

- name: Restart cockpit
  service:
    name: "{{ __cockpit_daemon }}"
    # noqa args[module]
    state: "{{ 'restarted' if cockpit_started | bool else 'stopped' }}"
```
not sure how ALP-Dolomite works - it will not perform these actions upon reboot?

Reply:
A reboot is required after installing new packages due to the use of transactional updates, the 'Flush handlers' task is redundant I'll update the changes